vulnerabilities/api.py

```diff
@@ -254,6 +254,24 @@ def bulk_search(self, request):
 
         return Response(response)
 
+    @action(detail=False, methods=["get"])
+    def all(self, _):
```

Comment:
Could you explain this change? I see the `request` arg is not used in this method, but that's a bad idea to rename function parameter with `_` imo.